<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signupForm</class>
 <widget class="QDialog" name="signupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create your account</string>
  </property>
  <property name="windowIcon">
   <iconset resource="LIBPRO.qrc">
    <normaloff>:/icon/Resources/icon/LP icon.png</normaloff>:/icon/Resources/icon/LP icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{font-family: &quot;Ubuntu Light&quot;; font-size: 15px; background-color: white; color:#422b25}
QLabel{font-family: &quot;Ubuntu Light&quot;; color: white;}
QFrame{background-color:rgba(39,62,46, 0.2); border-radius: 3px;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>361</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#signup
{
background-color: rgb(98, 175, 229);
border: 0px;
border-radius: 2px;
font-size: 20px;
font-family: &quot;Ubuntu Light&quot;;
color: white;
}
#signup:hover
{
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #1496bb, stop: 1 rgb(46, 150, 223));
border: 0px;
border-radius: 2px;
font: 25 20px &quot;Ubuntu Light&quot;;
color: white;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
background-color: transparent;
</string>
    </property>
    <property name="text">
     <string>Account name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="accountName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 1px solid rgb(250, 250, 250);
border-left: 1px solid rgb(250, 250, 250);
border-right: 1px solid rgb(250, 250, 250);
border-bottom: 1px solid gray;
border-radius:2px;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 1px solid rgb(250, 250, 250);
border-left: 1px solid rgb(250, 250, 250);
border-right: 1px solid rgb(250, 250, 250);
border-bottom: 1px solid gray;
border-radius:2px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;


 background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
 background-color: transparent;
</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 1px solid rgb(250, 250, 250);
border-left: 1px solid rgb(250, 250, 250);
border-right: 1px solid rgb(250, 250, 250);
border-bottom: 1px solid gray;
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 1px solid rgb(250, 250, 250);
border-left: 1px solid rgb(250, 250, 250);
border-right: 1px solid rgb(250, 250, 250);
border-bottom: 1px solid gray;
border-radius:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
 background-color: transparent;

</string>
    </property>
    <property name="text">
     <string>Confirm password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SIGN UP</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(39,62,46, 1)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
color: white;
font-family: &quot;Ubuntu Light&quot;;
 background-color: transparent;

</string>
     </property>
     <property name="text">
      <string>Create a new account</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font-weight: 500;
text-decoration: underline;
font-family: &quot;Ubuntu Light&quot;;
color: white;
 background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: transparent;
border-top: 2px solid white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: transparent;
border-top: 2px solid white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>600</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_3{
background-color:rgba(39,62,46, 1)
}
#login
{
border-radius: 2px;

	background-color: rgb(66, 162, 97);
font-family:&quot;Ubuntu Light&quot;;
font-size: 15px;
color: white;
}
#login:hover
{
background-color: rgb(37, 103, 55);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>5</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
color: white;
background-color: tranparent;
background-color: rgba(104, 120, 130, 0);</string>
     </property>
     <property name="text">
      <string>Already have an account?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login">
     <property name="geometry">
      <rect>
       <x>274</x>
       <y>10</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
background-color: transparent;
</string>
    </property>
    <property name="text">
     <string>Identification:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="identification">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 1px solid rgb(250, 250, 250);
border-left: 1px solid rgb(250, 250, 250);
border-right: 1px solid rgb(250, 250, 250);
border-bottom: 1px solid gray;
border-radius:2px;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0.7);
border: 2px solid rgb(235, 76, 75);
font-size: 15px;
border-radius: 4px;
color: red;
font-family: &quot;Ubuntu Light&quot;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(235, 76, 75);
font-size: 10px;
background-color: white;
border-radius: 4px;
color: red;
font-family: &quot;Ubuntu Light&quot;</string>
    </property>
    <property name="text">
     <string>greater than 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(235, 76, 75);
font-size: 10px;
border-radius: 4px;
color: red;
font-family: &quot;Ubuntu Light&quot;;
background-color: white;
</string>
    </property>
    <property name="text">
     <string>greater than 5</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>identification</tabstop>
  <tabstop>email</tabstop>
  <tabstop>accountName</tabstop>
  <tabstop>password</tabstop>
  <tabstop>confirm</tabstop>
  <tabstop>signup</tabstop>
  <tabstop>login</tabstop>
 </tabstops>
 <resources>
  <include location="LIBPRO.qrc"/>
 </resources>
 <connections/>
</ui>
